### DISCUSSION TITLE
How can i specify an ingressClassName

### DISCUSSION THREAD

**Discussion by lordigon** [arkmq-org/arkmq-org-broker-operator#630] [Q&A] [answered]
I tried to use your Operator on a real k8s cluster on AWS but I saw within the ingress resource created that the ingress class is missing.
Is it possible to specify an ingressClassName as usually done when you define your own ingress?

Best regards
Flavio

**Comment by gaohoward** [comment, marked as answer]
The ingressClassName is not exposed by the operator. Can you raise the feature issue pls?

**Reply by brusdev** [reply]
@howardgao @lordigon an issue already exists #557, feel free to add your comment to it
